docs/concepts.md

```diff
@@ -1,15 +1,98 @@
-# TriggerMesh concepts
+The following diagram depicts the core concepts of the TriggerMesh Cloud-Native Integration Platform: Source, Target, Function, Transformation, Filter, Splitter.
 
-## Source
+![](./assets/images/concepts.png)
+
+TriggerMesh allows you to integrate applications (on-premises or cloud services) by defining event sources and targets which are seen as the start and the end of a so-called Bridge. While events flow through the Bridge they can undergo splitting, filtering and transformation. Transformation of events can be done declaratively or through a function.
+
+To route events TriggerMesh makes use of the Knative eventing primitives (e.g Brokers, Channels, Triggers).
+
+Each component of a Bridge can be defined using an API object available through the TriggerMesh platform or through Knative.
```

Comment:
Do we usually communicate that the "TriggerMesh Platform" and "Knative" are two separate components, or that "TriggerMesh Platform" encompasses Knative, and everything else it depends on?

This is not a request for change, but I've seen both so it might be a good question to ask ourselves before merging.

Reply:
That is true. My comment was more on the highlighted sentence, "an API object available through TriggerMesh **or** Knative", which I was afraid may make them sound like two separate things.